Forge.TreeWalker/src/TreeWalkerSession.cs

```diff
@@ -56,6 +54,18 @@ public class TreeWalkerSession : ITreeSession
         /// </summary>
         public static string IntermediatesSuffix = "_Int";
 
+        /// <summary>
+        /// The TreeInput suffix appended to the end of the key in forgeState that maps to this tree walking session's TreeInput object.
+        /// </summary>
```

Comment:
nit:     better if you have the example as the existing comments like
 /// Key: <SessionId>_CTN
or 
  /// Key: <SessionId>_<TreeActionKey>_Int